### ISSUE TITLE
task: flask-cors dependency is needed to allow cross-origin requests

### ISSUE THREAD

**Issue by mtreacy002** [anitab-org/bridge-in-tech-backend#51]
**Describe the task**
To allow multiple servers to run locally when testing Bridge in tech functionalities such as register and login, 3 servers (BIT and MS backend and BIT React frontend) need to run on local machine and sending/receiving requests from each other.

<img width="731" alt="Screen Shot 2020-06-10 at 8 28 12 pm" src="https://user-images.githubusercontent.com/29667122/84591392-4f241380-ae81-11ea-831f-a71a3d14536e.png">

**Acceptance Criteria**

- [ ] It should not have the error message above. Instead, BIT should be able to send API call successfully

**Additional context**
- Need this to be approve for Register functionality on BIT-UI [PR#18](https://github.com/anitab-org/bridge-in-tech-web/issues/18) can be tested.

**Comment by meenakshi-dhanani** [comment]
You might need to reword this to a task. It's not a bug. This is expected behavior if you attempt cross origin requests. I'm going to do some renaming. Removed the screenshot of a bad request, it's slightly misleading. A bug is something you discover post development. When you're developing everything you need to make it work is a task, story, etc.
